### DISCUSSION TITLE
Publishing new version

### DISCUSSION THREAD

**Discussion by xolotlatoani** [scullyio/scully#1388] [Q&A] [answered]
After merging a PR, how long does one need to wait ?

Talking more about: #1371

**Comment by SanderElias** [comment]
Releasing the 'latest' versions is manual. If you want to play with the new stuff, you can always use the `@develop` version. Those are put out as a step in our testing procedure, even _before_ something is merged into master. 
You can use those by doing `npm i @scullyio/scully@develop @scyllyio/nglib@develop` in your project.

(If you find a version that works for you, you might want to fix the exact version in your `package.json` as its part of our testing procedure, it is not the most stable option ;) )

**Reply by xolotlatoani** [reply]
Thanks for the reply!

Is there a schedule for when you guys manually release the newest versions?

**Reply by SanderElias** [reply]
No, we are dependent on available capacity, and also we have quite a few 3rth parties we have to take into account.
I would love to have at least a monthly update, and I think that will be possible now again.
We were kept in a catch 22 for a while because decisions had to be made about backward compatibility. There were some serious breaking changes in TS that made this hard, and to top that off, IVY did come with a new way of distributing libraries. 
Angular V12 does enable us to switch over to that new way (we will do that soon!) and our hope is that a situation like this will not happen again.

**Comment by Jefiozie** [comment, marked as answer]
Hi everyone, there is a new version available for you.
For the changes you can look at our tag, [here](https://github.com/scullyio/scully/tree/v2.0.1) and on npm [here]( https://www.npmjs.com/package/@scullyio/scully/v/2.0.0).

We would appreciate any feedback. 

Thank you 🙏🏼

**Comment by SanderElias** [comment]
An update here. We now moved (a while ago actually) to an automated system, that will create a release, when we merge into `main`.
This way you can view our release notes as a changelog, and we keep the flow fully automated.

The release to NPM happens automatically when we release a `draft release`. 
Because we also have tooling and docs updates, sometimes we do 'skip' a GitHub release and merge them into the next version.
